pkg/reconciler/ingress/ingress.go

```diff
@@ -25,6 +25,18 @@ type Reconciler struct {
 	netclient     netclientset.Interface
 }
 
+const (
+	preferHeaderField               = "Prefer"
+	preferAsyncValue                = "respond-async"
+	preferSyncValue                 = "respond-sync"
+	asyncFrequencyTypeAnnotationKey = "async.knative.dev/frequency.type"
+	asyncFrequencyType              = "always.async.knative.dev"
+	publicLBDomain                  = "istio-ingressgateway.istio-system.svc.cluster.local"
```

Comment:
These aren't guaranteed but I think this is temporary anyway?

Reply:
correct, we'll get it from child ingress.